src/elexmodel/models/BaseElectionModel.py

```diff
@@ -264,3 +267,46 @@ def get_coefficients(self):
         These coefficients are for the point prediciton only, not for the lower or upper intervals models.
         """
         return self.features_to_coefficients
+
+    def compute_lambda(
+        self,
+        reporting_units,
+        possible_lambda_values=[],
+        estimand="",
+        K=3,
+    ):
+        if (
+            len(self.features) == 0
+            or len(possible_lambda_values) == 0
+            or estimand == ""
+            or not (f"{estimand}" in reporting_units)
+        ):
+            return 0, 0
+
+        MAPE_arr = np.full_like(possible_lambda_values, 0)  # array of MAPE sums for each lambda
```

Comment:
this is totally fine as is, but `zeros_like` does this without having to set the value.